src/app/utils/dim-error.ts

```diff
@@ -14,7 +14,7 @@ export class DimError extends Error {
   cause?: Error;
 
   /** Pass in just a message key to set the message to the localized version of that key, or override with the second parameter. */
-  constructor(messageKey: string, message?: string) {
+  constructor(messageKey: I18nKey, message?: string) {
```

Comment:
I feel like this should be something like this as to not require a non-localized code to be present in the localization files. 

```suggestion
  /** Pass in just a message key to set the message to the localized version of that key */
  constructor(messageKey: I18nKey);
  /** Use the key as an internal code and provide a localized message instead */
  constructor(messageKey: string, message: string);

  constructor(messageKey: I18nKey, message?: string) {
```